### PR TITLE
Solution for #14

### PR DESCRIPTION
**Pull request by pkdotson**
Find diagonal element in 2d array that is always cubed.

### CODE REVIEW

File: solutions/14.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// finds number in diagnalArray always cubed.
+// Phillip Kelley-Dotson
+
+const diagnalArray = (Array) => {
```
Comment (msach22) [inline solutions/14.js]: @pkdotson please rename to solution, export as solution, and import as solution on the test file

Comment (msach22) [inline solutions/14.js]: you can see other files for example :)

Comment (pkdotson) [inline solutions/14.js]: Done

File: test/14.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const expect = require('chai').expect;
+const diagnal = require('../solutions/14');
+const solution = diagnal.solution;
+// const yourSolution = "";
+
```
Comment (msach22) [inline test/14.js]: let solution = require('../solutions/14').solution;
//solution = require('../yourSolution').solution;

and remove the //const yourSolution line
try this method!

File: test/14.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+describe('Diagnal array', () => {
+  it('should return array of numbers diagnal to eatch', () => {
+    const Array = [[1,2,3],[2,3,4],[3,4,5]];
+    const expected = [1,3,5];
```
Comment (msach22) [inline test/14.js]: name variables using camel case

array instead of Array

File: test/14.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+// solution = require('../yourSolutionl/14').solution;
+
+
+describe('Diagnal array', () => {
+  it('should return array of numbers diagnal to eatch', () => {
+    const array = [[1,2,3],[2,3,4],[3,4,5]];
+    const expected = [1,3,5];
+    const actual = solution(array);
+    expect(actual).eql(expected);
+  });
```
Comment (msach22) [inline test/14.js]: expect(solution(array)).eql(expected);

so you dont need to use actual :)

File: solutions/14.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// finds number in diagnalArray always cubed.
+// Phillip Kelley-Dotson
+
+const solution = (array) => {
+    let nuArry = [];
+    let c = 0;
+  for (let i=0; i<array.length; i++){
```
Comment (songz) [inline solutions/14.js]: indentation

**Comment by songz** [comment]
closing this due to inactivity.

**Pull request closed without merging**